primenet.py

```diff
@@ -851,6 +855,11 @@ def setup():
         options.memory = memory
         config.set(SEC.PrimeNet, "memory", str(memory))
         options.day_night_memory = memory * 0.9
+    if tf1g:
```

Comment:
You would need to provide a default value for `tf1g`, as otherwise this will raise an exception for non mfaktc/mfakto users.